<launch>

<node pkg="rosbag" type="play" name="player" output="screen" args="--clock '$(find wm_navigation)/bags/mapping_robolab_real_2015-10-05-10-05-32.bag'"/>

	<node pkg="wm_navigation" type="wm_map_server_node" name="wm_map_server" output="screen" args="'$(find wm_navigation)/maps/lab.pcd'">
		<param name="frame_id" type="string" value="/map" /> 
		<param name="mapping" type="boolean" value="false" />
		<param name="pointcloud_min_z" type="double" value="-0.5" /> 
		<param name="pointcloud_max_z" type="double" value="2.0" />
		<param name="pointcloud_max_x" type="double" value="4.0" />
		<param name="pointcloud_max_t_xy" type="double" value="0.3" />
		<param name="pointcloud_max_t_xz" type="double" value="0.4" />
		
		<remap from="cloud_in" to="/camera/depth_registered/points" />
	</node>


 <node pkg="wm_navigation" type="wm_localization_node" name="wm_localization" output="screen">
  		<param name="frame_id" type="string" value="map" />
 		<param name="pointcloud_min_z" type="double" value="-0.5" /> 
		<param name="pointcloud_max_z" type="double" value="2.0" />
		<param name="pointcloud_max_x" type="double" value="4.0" />
		<param name="pointcloud_max_t_xy" type="double" value="0.3" />
		<param name="pointcloud_max_t_xz" type="double" value="0.4" />
		<remap from="cloud_in" to="/camera/depth_registered/points" />
  	</node> 


	<node pkg="rviz" type="rviz" output="screen"
		name="rviz"
		args="-d $(find wm_navigation)/rviz/wm_localization.rviz"/>

</launch>
